utils/firestore.js

```diff
@@ -1506,12 +1506,54 @@ const setPackageReceiptUSPS = async (data) => {
 
 const setPackageReceiptFedex = async (data) => {
     try {
-        const snapshot = await db.collection("boxes").where('959708259', '==', data.scannedBarcode).get();
+        let conversion = {
```

Comment:
I would declare this in the ``shared.js`` file so it can be referenced in the future.